### DISCUSSION TITLE
Welcome to 5126CenterStage-roadrunnerProject Discussions!

### DISCUSSION THREAD

**Discussion by aaronvesperman** [AnkenyRoboticsClub/5126CenterStage-roadrunnerProject#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aaronvesperman** [comment]
@Haaduken Do you have any recommendations for creating a branch and preparing the team code for state?

 @SamaraF24  @Gabe2008 Let's try these suggestions from @Haaduken this week!

**Comment by Haaduken** [comment]
If everything on master is the latest that you used for the robot at the state qualifiers, I suggest making a github tag right there and label it so. From there, as you dev for state, branch off main and PR into it. I can help with any of that setup as well, just ping me!

**Reply by Gabe2008** [reply]
@SamaraF24 I made a new branch and added a tag for the code from the league meet.
